model/client4.go

```diff
@@ -112,17 +112,11 @@ func BuildResponse(r *http.Response) *Response {
 	}
 }
 
-// SetToken sets token e.g. Personal Access Token
 func (c *Client4) SetToken(token string) {
 	c.AuthToken = token
 	c.AuthType = HEADER_BEARER
 }
 
-func (c *Client4) MockSession(sessionToken string) {
```

Comment:
Although /our/ code no longer uses `MockSession`, anyone importing `github.com/mattermost/mattermost-server/model` might still be using it, and removing a function amounts to semver breakage. Can we keep this function around, but just have it call the new `SetToken` instead? Might even be good to add a GoDoc:
```go
// MockSession is deprecated in favour of SetToken
func (c *Client4) MockSession(sessionToken string) {
```

Reply:
I didn't really understand semver. Fixed.